spec/ical_filter_proxy_spec.rb

```diff
@@ -7,6 +7,7 @@
 
   describe '.config' do
     before do
+      ENV['ICAL_FILTER_PROXY_API_KEY'] = "abc12"
       expect(IcalFilterProxy).to receive(:config_file_path).and_return File.expand_path('../config.yml.example', __dir__)
     end
 
```

Comment:
It's implicit from the modification you've made, but I'd appreciate an explicit "It interpolates values from ENV test"

Reply:
... and as part of that ensure we're not leaking other parts of ENV into our config

Reply:
Can change, yes

Reply:
Done.

Reply:
Changed. I moved config's for unittest into fixture folder so that we do not use the example config in tests. might be confusing if tests fail just due to adjusting example config.